tools/flash_shelly.sh

```diff
@@ -51,21 +63,30 @@ function check_brew {
 
   echo -e '\033[1mInstalling brew...\033[0m'
   echo -e '\033[1mPlease follow instructions...\033[0m'
-  echo -e '\033[1mYou will be asked for your password to install breww...\033[0m'
+  echo -e '\033[1mYou may be asked for your password...\033[0m'
   echo -e '\033[1m \033[0m'
   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
 }
 
 if [ "$(which timeout 2>/dev/null)" == "" ]; then
-  check_brew
   echo -e '\033[1mInstalling coreutils...\033[0m'
-  brew install coreutils
+  echo -e '\033[1mYou may be asked for your password...\033[0m'
+  echo $($installer coreutils 2>/dev/null)
 fi
 
 if [ "$(which jq  2>/dev/null)" == "" ]; then
-  check_brew
+  check_installer
   echo -e '\033[1mInstalling jq...\033[0m'
-  brew install jq
+  echo -e '\033[1mYou may be asked for your password...\033[0m'
+  echo $($installer jq 2>/dev/null)
```

Comment:
don't redirect stderr to dev null, in case errors happen user should be able to see them. it'll be a bit more noise on the firs run, but that's ok.

Reply:
ok

Reply:
https://github.com/mongoose-os-apps/shelly-homekit/pull/118/commits/1d42a9f35a8ea6543e47e75c7f28b9ed28baac8e